### PR TITLE
Cleans icons of unused sprites, unused code.

### PR DESCRIPTION
**Pull request by Kurgis**
## About The Pull Request

The pull requests cleans out unused icons from many of the .dmi files. Deletes only icons that are both:
A) Of poor quality not fitting the guideline, and thus unlikely to be reused.
B) Are not used within the code.

Removes old border firedoor object, and related procs.

## Why It's Good For The Game

General decluttering, otherwise it should have minimal affect in-game.

## Changelog
:cl:
del: Removed Bay news articles from the news article program.
imagedel: Deleted many icons unused in code which were of poor quality.
imagedel: Deleted news article images.
/:cl:

### CODE REVIEW

File: code/_onclick/adjacent.dm [hunk 1]
```diff
@@ -105,22 +105,6 @@ Quick adjacency (to turf):
...
-	so they can be interacted with without opening the door.
-*/
-/obj/machinery/door/Adjacent(var/atom/neighbor)
-	var/obj/machinery/door/firedoor/border_only/BOD = locate() in loc
-	if(BOD)
-		BOD.throwpass = TRUE // allow click to pass
-		. = ..()
-		BOD.throwpass = FALSE
-		return .
-	return ..()
```
Comment (drexample) [inline code/_onclick/adjacent.dm]: What is this?

File: code/game/machinery/doors/firedoor.dm [hunk 2]
```diff
@@ -366,44 +366,6 @@
...
-	update_nearby_tiles(need_rebuild)
-		var/turf/simulated/source = loc
-		var/turf/simulated/destination = get_step(source,dir)
-
-		update_heat_protection(loc)
-
-		if(istype(source)) SSair.tiles_to_update += source
-		if(istype(destination)) SSair.tiles_to_update += destination
-		return 1
-*/
```
Comment (drexample) [inline code/game/machinery/doors/firedoor.dm]: ????????

File: maps/_map_shims.dm [hunk 3]
```diff
@@ -49,3 +49,6 @@
 
 /obj/item/weapon/gun/energy/cassad
 	parent_type = /obj/item/weapon/gun/energy/pulse/cassad
+
+/obj/machinery/door/firedoor/border_only
```
Comment (drexample) [inline maps/_map_shims.dm]: ditto

**Comment by Kurgis** [comment]
I'm going to answer your three reviews together in this comment. It's an old firedoor object that has been disabled due to causing problem with ZAS. It's essentially a windoor which is also a firedoor. Of course, I'm sure it's handled differently, otherwise it wouldn't have been disabled due to causing problems with the atmos system. Checking back through Baystation is appears that they also removed this code remnant themselves in  a recent PR: https://github.com/Baystation12/Baystation12/pull/26369

Now, the questions of why it exists in the map, and I have good explanation. When the border_only firedoor was removed, the only removed its code, not the object. This would cause border_only to inherit everything from firedoor, and would allow whoever made the changes to not have to use map editing. When our map was created, they likely took active objects from the Torch map, and then used them into the Eris map, causing this old object to be placed everywhere.

**Comment by drexample** [comment]
> 
> 
> I'm going to answer your three reviews together in this comment. It's an old firedoor object that has been disabled due to causing problem with ZAS. It's essentially a windoor which is also a firedoor. Of course, I'm sure it's handled differently, otherwise it wouldn't have been disabled due to causing problems with the atmos system. Checking back through Baystation is appears that they also removed this code remnant themselves in a recent unrelated PR: [Baystation12/Baystation12#26369](https://github.com/Baystation12/Baystation12/pull/26369)
> 
> Now, the questions of why it exists in the map, and I have good explanation. When the border_only firedoor was removed, the only removed its code, not the object. This would cause border_only to inherit everything from firedoor, and would allow whoever made the changes to not have to use map editing. When our map was created, they likely took active objects from the Torch map, and then used them into the Eris map, causing this old object to be placed everywhere. I've created an issue for this, and SpadesSlick is aware of it: #3692

Then reflect it in PR's description.

**Pull request merged**